### PR TITLE
Allow rollbacks to an arbitrary release label

### PR DESCRIPTION
**Pull request by silhouettes**
Adding support as per #83. Relies on server-side support which has been implemented!

### CODE REVIEW

File: cli/script/command-parser.ts [hunk 1]
```diff
@@ -185,10 +185,12 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
...
     })
     .command("rollback", "Performs a rollback on the latest package of a specific deployment", (yargs: yargs.Argv) => {
-        yargs.usage(USAGE_PREFIX + " rollback <appName> <deploymentName>")
+        yargs.usage(USAGE_PREFIX + " rollback <appName> <deploymentName> [--targetRelease <releaseLabel>]")
             .demand(/*count*/ 3, /*max*/ 3)  // Require exactly three non-option arguments.
-            .example("rollback MyApp Production", "Perform a rollback on the \"Production\" deployment of \"MyApp\"");
-            
+            .example("rollback MyApp Production", "Perform a rollback on the \"Production\" deployment of \"MyApp\"")
+            .example("rollback MyApp Production --targetRelease v4", "Perform a rollback on the \"Production\" deployment of \"MyApp\" to the v4 release.")
+            .option("targetRelease", { alias: "r", default: null, demand: false, description: "The label of the release to be rolled back to", type: "string" });
```
Comment (shishirx34) [inline cli/script/command-parser.ts]: do you think it is intuitive to find the release label? or may be put in the command to list the release labels in the description here?

File: sdk/script/management/account-manager.ts [hunk 2]
```diff
@@ -749,10 +749,10 @@ export class AccountManager {
         });
     }
 
-    public rollbackPackage(appId: string, deploymentId: string): Promise<void> {
+    public rollbackPackage(appId: string, deploymentId: string, targetRelease?: string): Promise<void> {
         return Promise<void>((resolve, reject, notify) => {
             var requester = (this._authedAgent ? this._authedAgent : request);
-            var req = requester.post(this.serverUrl + "/apps/" + appId + "/deployments/" + deploymentId + "/rollback");
+            var req = requester.post(this.serverUrl + "/apps/" + appId + "/deployments/" + deploymentId + "/rollback/" + (targetRelease || ""));
```
Comment (shishirx34) [inline sdk/script/management/account-manager.ts]: I prefer having this as a query string since it is optional.

**Comment by shishirx34** [comment]
LGTM!

File: cli/script/command-parser.ts (hunk 1 above)
Comment (lostintangent) [inline cli/script/command-parser.ts]: I think of this as an edge case scenario, so I think it's fine to not overly document how to get the label. That said, maybe the description can have an "(e.g. v12)" example to make it a little clearer.

**Comment by lostintangent** [comment]
LGTM!

**Pull request merged**